core/math/a_star.cpp

```diff
@@ -257,14 +257,14 @@ void AStar::reserve_space(int p_num_nodes) {
 	points.reserve(p_num_nodes);
 }
 
-int AStar::get_closest_point(const Vector3 &p_point, bool ignore_disabled) const {
+int AStar::get_closest_point(const Vector3 &p_point, bool include_disabled) const {
```

Comment:
Parameters should start with `p_`, so this should be `p_include_disabled`.

Reply:
Fixed.